docs/react-native/docs/triggers.md

```diff
@@ -143,16 +143,34 @@ const trigger: TimestampTrigger = {
   //...
   alarmManager: {
     allowWhileIdle: true,
-  }
+  },
 };
 ```
 
 Please note, for iOS, a repeating trigger does not work the same as Android - the initial trigger cannot be delayed:
+
 - `HOURLY`: the starting date and hour will be ignored, and only the minutes and seconds will be taken into the account. If the timestamp is set to trigger in 3 hours and repeat every 5th minute of the hour, the alert will not fire in 3 hours, but will instead fire immediately on the next 5th minute of the hour.
 - `DAILY`: the starting day will be ignored, and only the time will be taken into account. If it is January 1 at 10 AM and you schedule a daily recurring notification for January 2 at 11 AM, it will fire on January 1 at 11 AM and every day thereafter.
 - `WEEKLY`: the starting week will be ignored, and only the day and time will be taken into account.
 
->  For more details, please see the discussion [here](https://github.com/notifee/react-native-notifee/issues/241).
+> For more details, please see the discussion [here](https://github.com/notifee/react-native-notifee/issues/241).
+
+### Android 12 Limitations
+
+Starting from Android 12, timestamp triggers cannot be created unless user specfically allow the [exact alarm permission](https://developer.android.com/reference/android/Manifest.permission#SCHEDULE_EXACT_ALARM). Before you create a timestamp trigger, check whether `SCHEDULE_EXACT_ALARM` permission is allowed by making a call to `getNotificationSettings`. If ` allowedTimestampTrigger` is `false`, you should educate the user on this permission and ask to allow scheduling alarms. You can then use `openAlarmPermissionSettings` function to display the Alarms & Reminder settings of your app.
+
+```js
+const settings = Notifee.getNotificationSettings();
+if (settings.android.allowedTimestampTrigger) {
```

Comment:
@vincent-paing can we rename this to `alarm` or something more generic? The reason being is that in the future other trigger types e.g. the `IntervalTrigger` could be created with the alarm manager.

Reply:
@helenaford I will rename it to `alarm` to be consistent with naming for `iOSSettings`